src/index.html

```diff
@@ -8,6 +8,33 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="card" data-qa="card">
+      <div class="card__image-container">
+        <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+        >
+      </div>
+
+      <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <span class="card__code">Product code: 195434</span>
+
+      <div class="stars stars--4">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <span class="stars__number">Reviews: 5</span>
+      </div>
```

Comment:
`stars__number ` is not an element of `stars` block
```suggestion
      <div class="card__rating">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        <p class="card__reviews">Reviews: 5</p>
      </div>
```